salt/modules/rh_service.py

```diff
@@ -97,6 +97,15 @@ def __virtual__():
                     'RedHat-based distros >= version 7 use systemd, will not '
                     'load rh_service.py as virtual \'service\''
                 )
+        if __grains__['os'] == 'Amazon':
+            if int(osrelease_major) in (2016, 2017):
+                return __virtualname__
+            else:
+                return (
+                    False,
+                    'Amazon Linux >= version 2 use systemd, will not '
```

Comment:
`use` --> `uses` and then maybe let's split the message up into 2 sentences with `systemd. Will not`.